MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,593 @@
+
+ * **FromFile**: Required to specify the path of the file.
+ * **FromResource**: Required to set image as embedded resource.
+ * **FromStream**: Required to load image from byte[] array.
+ * **FromURI**: Required to set image from a web service or website.
+
+To load image (embedded resource) in DataGridImageColumn, follow the code example:
+ 
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+<syncfusion:SfDataGrid x:Name="dataGrid" ItemsSource="{Binding OrderInfoCollection}"  ColumnWidthMode="Fill" utoGenerateColumnsMode="None">
+    <syncfusion:SfDataGrid.Columns >
+        <syncfusion:DataGridImageColumn  MappingName="DealerImage" HeaderText="Dealer Image" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+![DataGrid with image column](Images\column-types\maui-datagrid-column-imagecolumn.png)
+### Aspect
+
+SfDataGrid allows you to set the [Aspect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridImageColumn.html#Syncfusion_Maui_DataGrid_DataGridImageColumn_Aspect) to size the loaded images within the bounds of the grid cell (whether to stretch, crop or letterbox) using the `DataGridImageColumn.Aspect` property. The supported aspects are described below, the default value is AspectFit.
+
+* **AspectFill**: Clips the image so that it fills the display area while preserving the aspect (no distortion).
+
+* **AspectFit**:  Letterboxes the image (if required) so that the entire image fits into the display area, with blank space added to the top/bottom or sides depending on whether the image is wide or tall.
+
+* **Fill**: Stretches the image to completely and exactly fill the display area. This may result in the image being distorted.
+
+* **Center**: Places the image to the center of the cell.
+
+To set `Aspect` to images loaded inside `DataGridImageColumn`, refer the below code snippet.
+
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   AutoGenerateColumnsMode="None"
+                   ItemsSource="{Binding OrdersInfo}">
+    <syncfusion:SfDataGrid.Columns>
+        <sfGrid:DataGridImageColumn MappingName="DealerImage" Aspect="AspectFit"/>
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+## DataGridTemplateColumn
+
+The DataGridTemplateColumn is derived from DataGridColumn. Hence, it inherits all the properties of DataGridColumn. It allows you to extend the functionality of DataGridColumn with own view by creating the [CellTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridTemplateColumn.html#Syncfusion_Maui_DataGrid_DataGridTemplateColumn_CellTemplate).
+
+The following table provides the list of properties in DataGridTemplateColumn:
```

Comment:
@NirmalKumarYuvaraj  - This is not required.

Reply:
Removed the content.